addons/zeus/CfgEventHandlers.hpp

```diff
@@ -4,3 +4,11 @@ class Extended_PreInit_EventHandlers {
         init = QUOTE(call COMPILE_FILE(XEH_preInit));
     };
 };
+
+class Extended_InitPost_EventHandlers {
+    class AllVehicles {
+        class ADDON {
+            init = QUOTE(call FUNC(addObjectToCurator));
```

Comment:
This will error as the function uses `params` and no argument is passed (`params` doesn't handle a `nil` input unless it's within an array)